components/tabcordion/src/styled.js

```diff
@@ -17,78 +17,62 @@ export const TabRow = forwardRef((props, ref) => (
 
 export const TabItem = ({ appearance, isJustified, isLast, isSelected, ...props }) => {
 	const { COLORS: colors } = useBrand();
-	const common = {
-		flex: isJustified ? 1 : 0,
-		fontSize: '1.6rem',
-		marginRight: '0.2rem',
-		outline: 0,
-		padding: '1.4rem 1.8rem',
-		textAlign: 'left',
-		textDecoration: 'none',
-		transition: 'background .3s ease',
-		width: '100%',
-
-		'&:last-child': {
-			marginRight: 0,
-		},
-	};
 	const styles = {
 		soft: {
 			backgroundColor: isSelected ? '#fff' : colors.background,
 			borderTopLeftRadius: 3,
 			borderTopRightRadius: 3,
 			border: `1px solid ${colors.border}`,
 			borderBottom: 0,
-			color: '#333',
-			cursor: 'pointer',
-			marginBottom: isSelected ? '-0.1rem' : null,
-			paddingBottom: isSelected ? '1.5rem' : null,
+			color: colors.neutral,
+			marginBottom: isSelected ? '-0.0625rem' : null,
+			paddingBottom: isSelected ? '0.9375rem' : null,
 		},
 		lego: {
 			backgroundColor: isSelected ? '#fff' : colors.hero.default,
 			border: `1px solid ${isSelected ? colors.border : 'transparent'}`,
 			borderBottom: 0,
 			color: isSelected ? colors.text : colors.hero.foreground,
-			cursor: 'pointer',
-			marginBottom: isSelected ? '-0.1rem' : '0.2rem',
-			paddingBottom: isSelected ? '1.7rem' : null,
+			marginBottom: isSelected ? '-0.0625rem' : '0.125rem',
+			paddingBottom: isSelected ? '1.0625rem' : null,
 		},
 	};
 
-	return <button css={{ ...common, ...styles[appearance] }} {...props} />;
+	return (
+		<button
+			css={{
+				flex: isJustified ? 1 : 0,
+				fontSize: '1rem',
+				marginRight: '0.125rem',
+				outline: 0,
+				padding: '0.875rem 1.125rem',
+				textAlign: 'left',
+				textDecoration: 'none',
+				transition: 'background .3s ease',
+				width: '100%',
+				cursor: 'pointer',
+
+				'&:last-child': {
+					marginRight: 0,
+				},
+				...styles[appearance],
```

Comment:
`appearance` should be `look` now: https://github.com/WestpacGEL/GEL/tree/develop#props-api-vocabulary